### PR TITLE
Allow tabs to be disabled on the visualize.jsp page

### PR DESCRIPTION
**Pull request by zheins**
# What? Why?
Allow tabs to be disabled for performance or other reasons.
# Notify reviewers
@jjgao @zhx828 

Signed-off-by: zheins <zackheins@gmail.com>

### CODE REVIEW

**Comment by jjgao** [comment]
@zheins any idea why Travis failed?

**Comment by zheins** [comment]
Hey @jjgao, it failed on something seemingly not related: 

`Tests run: 8, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 6.574 sec <<< FAILURE! - in org.mskcc.cbio.portal.scripts.TestImportTabDelimData
testImportmRnaData2(org.mskcc.cbio.portal.scripts.TestImportTabDelimData)  Time elapsed: 0.049 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<[9]> but was:<[NaN]>
	at org.junit.Assert.assertEquals(Assert.java:123)
	at org.junit.Assert.assertEquals(Assert.java:145)
	at org.mskcc.cbio.portal.scripts.TestImportTabDelimData.testImportmRnaData2(TestImportTabDelimData.java:348)`

I updated the branch and it seems to be passing now.

**Comment by zhx828** [comment]
@zheins The code looks good to me.

File: core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java [hunk 1]
```diff
@@ -715,6 +717,17 @@ public static String getDbVersion() {
...
     
+    public static List<String> getDisabledTabs() {
+        String disabledTabs = "";
+        try {
+            disabledTabs = properties.getProperty(DISABLED_TABS).trim();
+        }
+        catch (NullPointerException e) {}
+        
+        String[] tabs = disabledTabs.split("\\|");
+        return (tabs.length > 0 && disabledTabs.length() > 0) ? Arrays.asList(tabs) : new ArrayList<String>();
```
Comment (jjgao) [inline core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java]: maybe use hashset instead of list?

Comment (zheins) [inline core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java]: Oops looks like Ben merged before I saw this. @jjgao should we revert and I can make a new PR?

**Pull request merged**